src/index.js

```diff
@@ -65,14 +65,19 @@ function requireAuth(nextState, replace, callback) {
     });
 }
 
-function requireRoaster(nextState, replace) {
-    if (store.getState().roaster.roaster) {
+function requireRoaster(nextState, replace, callback) {
```

Comment:
@ghmeier just realized a lot of these cases weren't calling `callback()` too.  these should all work now, I think